SYCL/SpecConstants/2020/non_native/multiple-targets.cpp

```diff
@@ -0,0 +1,14 @@
+// REQUIRES: opencl-aot, cpu
+// UNSUPPORTED: cuda
+
+// RUN: %clangxx -fsycl -fsycl-targets=spir64,spir64_x86_64 %S/Inputs/common.cpp -o %t_spv_cpu.out
+// RUN: %CPU_RUN_PLACEHOLDER %t_spv_cpu.out
+// RUN: %GPU_RUN_PLACEHOLDER %t_spv_cpu.out
+// Ensure that image ordering does not impact the execution
```

Comment:
nit: I always find "image" kind of misleading to represent a "picture". `Binary image` or `device binary` would sound better for my subjective ear.

Reply:
Fair point, I agree this naming is a bit confusing. There are a few more places in the test suite that use it, though, so I'll submit a separate PR to make sure we use the "clear" naming everywhere.